be/src/exec/pipeline/query_context.cpp

```diff
@@ -114,22 +154,13 @@ QueryContextPtr QueryContextManager::get(const TUniqueId& query_id) {
 }
 
 void QueryContextManager::remove(const TUniqueId& query_id) {
-    size_t i = std::hash<size_t>()(query_id.lo) & QUERY_CONTEXT_MAP_SLOT_NUM_MASK;
+    size_t i = _slot_idx(query_id);
     auto& mutex = _mutexes[i];
     auto& context_map = _context_maps[i];
     auto& sc_map = _second_chance_maps[i];
 
     std::unique_lock<std::shared_mutex> write_lock(mutex);
 
-    // clean expired query contexts in sc_map
```

Comment:
Can we keep this cleanup procedure, and make the cleanup trigger both active and passively?

Reply:
ok